{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/forms/field.jinja

```diff
@@ -2,18 +2,22 @@
 {# or Sometimes third party controls come with their own styling, so for these widgets do not style #}
 {%-  macro input_widget_hook(field) -%}
     {%- if field.widget_type == "select"  -%}
-        select
+        input-box select
```

Comment:
Both select and input will need the input-box class so it can "look the part" of a form field, but radio button doesn't follow that spec so it isn't necessary here.